src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
```

Comment:
According to the checklist, we shouldn't use abstract classes to set behavior for classes. Consider using an interface instead.